src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/CrossClient/CrossClientCashoutAggregate.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+
+namespace Lykke.Job.BlockchainCashoutProcessor.Core.Domain
+{
+    public class CrossClientCashoutAggregate
+    {
+        public string Version { get; }
+
+        public CrossClientCashoutState State { get; private set; }
+
+        public DateTime StartMoment { get; }
+        public DateTime? OperationFinishMoment { get; private set; }
+
+        public Guid OperationId { get; }
+        public Guid ClientId { get; }
+        public string BlockchainType { get; }
+        public string BlockchainAssetId { get; }
+        public string HotWalletAddress { get; }
+        public string ToAddress { get; }
+        public decimal Amount { get; }
+        public string AssetId { get; }
+
+        public decimal? Fee { get; private set; }
+        public string Error { get; private set; }
+        public DateTime? MatchingEngineEnrollementMoment { get; private set; }
+        public Guid ToClientId { get; private set; }
+        public Guid CashinOperationId { get; private set; }
+
+        private CrossClientCashoutAggregate(
+            string version,
+            Guid operationId,
+            Guid clientId,
+            string blockchainType,
+            string blockchainAssetId,
+            string hotWalletAddress,
+            string toAddress,
+            decimal amount,
+            string assetId,
+            Guid toClientId,
+            Guid cashinOperationId,
+            DateTime? enrollmentDate = null,
```

Comment:
This ctor should be used to start new cashou only, so enrollmentDate and state should be removed from here.

Reply:
fixed